WalletWasabi.Gui/Assets/LegalDocuments.txt

```diff
@@ -33,23 +33,23 @@ We may make changes to these Terms, including when there are changes to our Serv
 
 2.1 DESCRIPTION OF SERVICES
 
-We strongly advise to only use our services if you are familiar with cryptocurrencies, especially Bitcoin. We recommend to first learn the basics of Bitcoin and Bitcoin transactions before starting to use our Services.
+We strongly advise you to only use our services if you are familiar with cryptocurrencies, especially Bitcoin. We recommend that you first learn the basics of Bitcoin and Bitcoin transactions before starting to use our Services.
 This Software consists of the Client Application and the Privacy Engine (both as defined below) and functions as a free, open-source non-custodial desktop wallet. The Software does not constitute an account where We or other third parties serve as financial intermediaries or custodians of Your bitcoin(s). The Software and our services are therefore exempt from the authority of the Gibraltar Financial Services Commission or any other financial institutions.
 While the Software has undergone beta testing and continues to be improved by feedback from the open-source user and developer community, We cannot guarantee there will not be bugs in the Software. You acknowledge that Your use of this Software is at Your own discretion and in compliance with all applicable laws. You are responsible for safekeeping Your passwords, private key pairs, PINs, Recovery Words, and any other codes You use to access the Software.
-We expressly declare that we do not store any private keys. Encrypted private key information is stored locally on your computer in a wallet file and/or backed up with Recovery Words, shown at the creation of the wallet. Private keys can be accessed with a password, you have given at the creation of the wallet. It is solely your responsibility to backup and to safekeep your private keys. In case of loss of private keys, you may permanently lose access to your public addresses, since we are not able to recover the private keys for You.
+We expressly declare that we do not store any private keys. Encrypted private key information is stored locally on your computer in a wallet file and/or backed up with Recovery Words, shown at the creation of the wallet. Private keys can be accessed with a password, which you have provided at the creation of the wallet. It is solely your responsibility to backup and to safekeep your private keys. In the event of your loss of private keys, you may permanently lose access to your public addresses, since we are not able to recover the private keys for You.
```

Comment:
GitHub shows that something is changed here, but I cannot tell what exactly...

Reply:
Before:
Private keys can be accessed with a password, **you have given** at the creation of the wallet.
**In case of loss of private keys**, you may permanently lose access to your public addresses, since we are not able to recover the private keys for You.

After:
Private keys can be accessed with a password, **which you have provided** at the creation of the wallet.
**In the event of your loss of private keys**, you may permanently lose access to your public addresses, since we are not able to recover the private keys for You.